calico-enterprise_versioned_docs/version-3.15/operations/monitor/prometheus/alertmanager.mdx

```diff
@@ -58,7 +58,7 @@ firing. These inhibition rules are defined in the alertmanager configuration
 file. You can define one by adding this configuration snippet to your
 `alertmanager.yaml`.
 
-```
+```yaml
```

Comment:
As I realize this needs a 'noValidataion' tag, I also realize that you may need to work through a considerable amount of merge conflicts between this PR and #583 . I don't know whether merging in a particular order will help. Be ready!

Reply:
I did not add the `noValidation` tag because the validator did not throw any errors. I'll add the tag in the other PR.

Yes, merge conflict is going to happen. I want to merge 583 first.